pepper-apis/src/test/java/org/broadinstitute/ddp/log/SlackAppenderTest.java

```diff
@@ -60,10 +57,4 @@ public void testSuccessfulLoggingToSlack() throws Exception {
             Assert.fail("Mock slack not running");
         }
     }
-
-    @After
-    public void shutDown() {
-        mockServerClient.stop();
```

Comment:
Shutting this down seems to cause issues with other tests that uses mockserver. There's no need to do a stop here since we're using `MockServerRule` which integrates with JUnit so JUnit will handle cleaning up those resources.

Reply:
I was wondering about this--thank you for tracking this down.